bundles/org.openhab.binding.oppo/src/main/java/org/openhab/binding/oppo/internal/communication/OppoReaderThread.java

```diff
@@ -42,7 +42,7 @@
      * @param connector the object that should handle the received message
      */
     public OppoReaderThread(OppoConnector connector, String uid) {
-        super(OppoBindingConstants.BINDING_ID + "-" + uid);
+        super("OH-binding-" + OppoBindingConstants.BINDING_ID + "-" + uid);
```

Comment:
You do not need the binding id as already included in uid (thing uid).

Reply:
The variable was named poorly so I corrected it. The idea was to help identify the particular connection (IP+ port or serial port name) in the case of multiple instances of the binding being used.

Reply:
Look at #8216, you are not following the new thread naming convention. You have to replace binding id by thing uid. Then having the serial port name as custom part in the name is ok.

Reply:
ok, it should be fixed now.